### PR TITLE
adds two new medkits

### PR DESCRIPTION
**Pull request by Bawhoppen**
:cl:
add: Medical Doctors now spawn with a special purple first-aid kit full of goodies.
add: Added emergency first-aid kits. They can spawn in emergency closets.
del: O2 medkits no longer spawn in emergency closets.
/:cl:

Two new medkits:
Emergency medkits spawn in emergency closets now, they have a few medi-pens of ephin, and tricord aswell as some gauze. 
Doctors also get a new purple kit with some patches and syringes of chems aswell as an 80u tricord auto-injector that CANNOT BE REFILLED.

### CODE REVIEW

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -61,6 +61,13 @@
 	volume = 100
 	list_reagents = list("nanites" = 80, "synaptizine" = 20)
 
+/obj/item/weapon/reagent_containers/hypospray/combat/tricord
+	name = "tricordrazine auto-injector"
+	desc = "A non-refillable basic healing injector filled with tricordrazine."
+	volume = 80
+	flags = null
```
Comment (AnturK) [inline code/modules/reagents/reagent_containers/hypospray.dm]: flags = NONE

**Comment by Shapsy** [comment]
but these purple medkits are shit compared to the white ones, you get an amazing 2 (count em, 2) patches compared to 24 brute packs/ointments which means you have to just take a normal medkit to do your job,

**Comment by Bawhoppen** [comment]
eh I don't know I did think about that but usually doctors already take a spare medkit

**Comment by Shapsy** [comment]
What? Why would someone take two medkits when one already lasts so long, and how does that justify burdening doctors with a useless one?

**Comment by MrStonedOne** [comment]
\>implying docters carry the white medkits at all.

**Comment by Bawhoppen** [comment]
@MrStonedOne white medkits are actually pretty great after I buffed ointment/bruisepacks

**Comment by MrStonedOne** [comment]
Ya, but that doesn't mean i ever remember or bother to grab one.

**Comment by Bawhoppen** [comment]
I usually get 2 white medkits and 1 brute if I'm going out

**Comment by MrStonedOne** [comment]
Baw, mang, I'm trying to help you out here, shh you.

**Comment by Bawhoppen** [comment]
happy-birthday-shining-purple-rose

**Comment by Bawhoppen** [comment]
git is suffering, why can't it just work with me

**Comment by Bawhoppen** [comment]
I just want to fucking hard reset why is it so hard

**Comment by PKPenguin321** [comment]
Stuffing emergency lockers with tricord feels pretty hugbox, if you'll excuse my buzzwords

**Comment by Bawhoppen** [comment]
@PKPenguin321 the tricord will barely heal you, maybe 20 damage at most per medipen, and theres only 2 tricord medipens in each

**Comment by Bawhoppen** [comment]
the travis is broken help

**Comment by Bawhoppen** [comment]
oh there it goes

**Comment by optimumtact** [comment]
Be patient, the web workers sometimes lag behind commits/pr's made.

**Comment by Davdichan** [comment]
Uh, 80u of tricord? Are these regular epens that inject everything at once or hypos that inject 10u at a time? 30u is OD on tricord.

**Comment by Niknakflak** [comment]
Yea, and the OD is pretty bad too.  Bad change.

**Comment by Ergovisavi** [comment]
Why not just give them a shitty hypospray synthesizer that only has 5 units of tricord at a time and slowly recharges? That way when someone stubs their toe, you can get them to full so they stop triggering you on your health HUD, and because it's only 5 units of weak tricord,  they're not stepping on Mediborg territory.

**Comment by TechnoAlchemisto** [comment]
Yeah don't put tricord in emergency closets, healing items really shouldn't be scattered around the station.

Just leave the emergency oxygen kits.

**Comment by Bawhoppen** [comment]
tricord is literally so shit

**Comment by Bawhoppen** [comment]
@Davdichan I'm not sure what you're saying, but the auto-injector is like the injectors ops/erts, it can inject at 10u at a time and this one cannot be refilled

**Comment by Shapsy** [comment]
this whole change is bad

**Comment by GeneralGoose** [comment]
again this person making changes to codebase to suit his own needs

**Comment by Bawhoppen** [comment]
nice catch 22, if I say that I'm not, I'm implying I don't play, if I say I do, then you win
niceme.me.exe

**Comment by Bawhoppen** [comment]
also who are you and what are you even talking about

**Comment by Davdichan** [comment]
So you made a nonrefillable hypo, autoinjectors dump all their chems in one go.

Also tricord being shit? Hue? Heals all damage types and that pen has more then enough to take someone from hard crit to green, power gamers are going to be looting these every round. Especially fucking wizards

**Comment by Bawhoppen** [comment]
@Davdichan every single thing (literally) you said is wrong
It doesn't dump all its chems at once, I even clarified in a comment earlier, Tricord only heals burn and brute VERY SLOWLY even at that, and if you want to waste all 80u on one person, you can easily heal them in countless other ways

**Comment by Davdichan** [comment]
@bawhoppen english isn't your first language or you're not reading my posts because !critism!

Tricord has a stupidly high vhance of healing each type of damage per tick, its not as fast or effective as synthflesh maybe but its still a catch all.

YOU are thinking of saline-glucose, which only has a.33% chance of healing brute and/or burn per tick. Which would be okay for a medipen or medkit item.

Putting potentially lethal amounts of chems into an item that can be used repeatedly is bad. This pen could be used as shank to od someone on tricord and likely knock them into crit after you figure the metab rate and click delay of a hypo-item.

**Comment by Bawhoppen** [comment]
You literally are not listening, unless Techno changed Tricord when it was readded, then Tricord is an extremely slow chem that heals only burn/brute, I don't even know what the fuck saline-glucose is because I don't fuck with the retarded goofchems

**Comment by Davdichan** [comment]
https://github.com/tgstation/tgstation/blob/a6ccb01821281f280b096e2bcb16887d43dee07f/code/modules/reagents/chemistry/reagents/medicine_reagents.dm
My phone can't copy text from the dm but you're seriously mixing up your chems. Search tricord then search saline. You aren't listening when people are telling you that you are using the wrong chem. Tricord has always been a fix all for damage types.

**Comment by Bawhoppen** [comment]
okay so it does heal all chem types? it is still as horribly slow as I said and has much better alternatives

**Comment by Davdichan** [comment]
80% chance per tick to heal one of each damage type. Metabolizes .4 units per tick, a 10u shot is 25 potential damage and an average of 20 healed across the board if RNG is playing by the rules. This makes it faster than omnizine with equal healing overall. In short one injection of that pen is 2 toolboxings AND a laser shot taken away. For a wizard, this is gods little gift. Fits in a pocket or a box, can be used instantly with the self channel time bruise packs and ointment have, and can be used on the move.

Offensively, This means a full pen WILL put someone into crit, its not as fast as a stun baton or fire axe but the clown can reasonably run up to a sec officer, stab them as many times as possible and then just wait for them to pass out to drag them off to maint to finish them off and steal items.

**Comment by Bawhoppen** [comment]
Okay so you can slowly heal someone? You can already heal them in other numerous ways? I am certain from empirical evidence that omnizine is far far better than tricord, I don't know if what you said is true or not. You seem to also forget about the OD part of it? You can't just amp yourself on tricord without reprecussions, and again, you could just use a bruisepack/ointment for much much faster healing.
In order to inject them to put them to sleep, you'd have to click up to 8 times as opposed to just a one stunprod and cablecuffs?

**Comment by PKPenguin321** [comment]
That isn't how you use question marks?

>you could just use a bruisepack/ointment for much much faster healing.

Those don't spawn in emergency lockers?

**Comment by Bawhoppen** [comment]
he was referring to the auto injector, the ones that spawn in emergency lockers are tiny

**Comment by KorPhaeron** [comment]
the station already has endless medical supplies we dont need more

ask the chemists if you want extra patches (you wont need extra because every vending machine has a ton and the storage room starts with a ton and cryo/sleepers exist anyway)

**Comment by Bawhoppen** [comment]
I don't care as much about the special kit, but I want the emergency lockers to have something useful and that also makes sense while not being OP, that is exactly what the emergency medkits were

**Pull request closed without merging**